core/src/main/java/com/radixdlt/consensus/epoch/EpochsConsensusModule.java

```diff
@@ -457,15 +458,17 @@ private VertexStoreFactory vertexStoreFactory(
       EventDispatcher<BFTInsertUpdate> updateSender,
       EventDispatcher<BFTRebuildUpdate> rebuildUpdateDispatcher,
       EventDispatcher<BFTHighQCUpdate> highQCUpdateEventDispatcher,
-      EventDispatcher<BFTCommittedUpdate> committedDispatcher,
       Ledger ledger,
-      Hasher hasher) {
+      Hasher hasher,
+      Serialization serialization,
+      Metrics metrics,
+      VertexStoreConfig vertexStoreConfig) {
     return vertexStoreState ->
         new VertexStoreAdapter(
-            VertexStoreJavaImpl.create(vertexStoreState, ledger, hasher),
+            new VertexStoreJavaImpl(
+                ledger, hasher, serialization, metrics, vertexStoreConfig, vertexStoreState),
```

Comment:
Probably best to call this `initialVertexStoreState` here, and in the `BftConfiguration` class.